starknet/src/space/space.cairo

```diff
@@ -288,7 +289,10 @@ mod Space {
             let mut proposal = self._proposals.read(proposal_id);
             assert_proposal_exists(@proposal);
             assert(proposal.author == author, 'Only Author');
-            assert(info::get_block_timestamp() < proposal.start_timestamp, 'Voting period started');
+            assert(
+                info::get_block_number().try_into().unwrap() < proposal.start_block_number,
```

Comment:
I think we're better off not using `.try_into().unwrap()` here and doing
`info::get_block_number < proposal.start_block_number.into()` ? will avoid having some match and panics

Reply:
yep changed